openstates/de/committees.py

```diff
@@ -22,29 +22,41 @@ def scrape(self, chamber, term):
 
         url = urls[chamber] % (session,)
         self.log(url)
-        page = lxml.html.fromstring(self.urlopen(url))
+        page = lxml.html.fromstring(self.get(url).text)
         page.make_links_absolute(url)
 
         committees = {}
 
-        for row in page.xpath('//td[@width="96%"]/table/tr[@valign="top"]'):
-            link = row.xpath('td/font/a[contains(@href, "opendocument")]')[0]
-            committees[link.text] = link.attrib['href']
-            self.log(link.attrib['href'])
-
-        for c in committees:
-            url = committees[c]
-            page = lxml.html.fromstring(self.urlopen(url))
-            page.make_links_absolute(url)
-            committee = Committee(chamber, c)
-            committee.add_source(url)
-
-            for tr in page.xpath('//td[@width="96%"]/table/tr'):
-                role_section = tr.xpath('td/b/font')
-                if(len(role_section) > 0):
-                    role = re.sub(r's?:$', '', role_section[0].text).lower()
-                    for member in tr.xpath('td/font/a'):
-                        name = re.sub('\s+', ' ', member.text)
-                        committee.add_member(name, role)
-
-            self.save_committee(committee)
+        for row in page.xpath('//tr'):
+            if len(row.xpath('./td')) > 0:
+                #if statement removes header tr
+                comm = row.xpath('.//a')[1]
+                comm_name = comm.text_content().strip()
```

Comment:
This is how they show up on the source (nothing has "committee" except that one) so I'm leaving it.